internal/backend/fixtures/fixtures.go

```diff
@@ -22,3 +22,5 @@ func NewBuiltinExecutor(xid sdktypes.ExecutorID, opts ...sdkmodule.Optfn) sdkexe
 	vs := kittehs.Must1(mod.Configure(context.TODO(), xid, sdktypes.InvalidConnectionID))
 	return sdkexecutor.NewExecutor(mod, xid, vs)
 }
+
+var BuiltinSchedulerConnectionID = kittehs.Must1(sdktypes.ParseConnectionID("con_3akt5cd1e00000000000000000"))
```

Comment:
why this specifc id? is it random?

Reply:
hardcoded. I'm using it also in tests and in there I cannot compare with randomID

Reply:
3kth